### PR TITLE
Feature/searchable dropdown

### PR DESCRIPTION
**Pull request by JonasNiestroj**
# Checklist (Only for frontend changes)
- [x] Changes tested in IE11
- [x] Responsive
- [x] Documented "How to use it"
- [x] Documented "How to install"
- [x] Featured in Styleguide

# Summary of this PR
I added a basic select component which uses the vue-multiselect library.
The component is customizable with the vue-multiselect options

### CODE REVIEW

**Comment by rotwurstesser** [comment]
Looks nice so far!

File: frontend/src/components/Select.vue [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<template>
+  <div>
+    <multiselect v-model="model" :options="options" v-bind="$attrs">
+    </multiselect>
+  </div>
+</template>
+
+<script>
+import Multiselect from 'vue-multiselect';
+export default {
```
Comment (schtibe) [inline frontend/src/components/Select.vue:10]: I'm unsure about this component. Same with input. We try to create those wrapper components to make things easier to use, but out in the field, I find that often those wrapper components don't provide all the flexibility we need. So we start extending those components, and soon we implement all the stuff of the child into the wrapper component. And we almost end up 

And in this one here, we don't actually add anything. It's good to have an example though!

I have an idea for the Input.vue to improve, will write this somewhere else.

**Review by schtibe — CHANGES_REQUESTED** [review]
There is quite some noise in this merge request, some formatting changes of the package.json and so on. Maybe we could sort this out by having another MR that fixes those things and then apply this one? I guess we can work more thoroughly here since we don't have a deadline ;)

File: frontend/styleguide.config.js [hunk 2]
```diff
@@ -38,6 +38,24 @@ const theme = {
   },
 };
 
+const transpileDependencies = [
+  'regexpu-core',
```
Comment (schtibe) [inline frontend/styleguide.config.js:42]: OK, weird, i checked out `master` and this stuff is already there...

File: frontend/styleguide.config.js [hunk 3]
```diff
@@ -87,6 +136,7 @@ module.exports = {
     path.join(__dirname, 'src/plugins/notifications.js'),
     path.join(__dirname, 'src/plugins/vuePortal.js'),
     path.join(__dirname, 'src/globals/svgIcon.js'),
+    path.join(__dirname, 'node_modules/svgxuse'),
```
Comment (schtibe) [inline frontend/styleguide.config.js:139]: This already is in master!!

**Pull request merged**